docs/data/material/getting-started/templates/sign-in-side/CustomIcons.tsx

```diff
@@ -10,63 +10,47 @@ export function SitemarkIcon({ sx }: IconProps) {
   return (
     <SvgIcon sx={sx}>
```

Comment:
Missing the removal of the `sx` prop/standardization of the design

Reply:
It was intentional there. If we remove the `sx` from it, it ends up too small.

<img width="1236" alt="Screenshot 2024-04-05 at 15 59 28" src="https://github.com/mui/material-ui/assets/37222944/1b9a387e-1b01-4013-b96e-fc5f9a0a3233">